app/controllers/application_controller.rb

```diff
@@ -2,4 +2,16 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  NAVIGATION_BUTTONS= [
+    { text: 'OSRA', href: 'root_path', path_regex: /^\/$/, glyph: 'glyphicon-home' },
```

Comment:
Removed. Neither it nor the "Dashboard" button have any value at the moment, but I'd like to still hold on to the idea of a functional dashboard with notifications etc., so it gets to stay :)